web/src/components/filtersform.js

```diff
@@ -109,12 +118,22 @@ class FiltersForm extends React.Component {
           <Form onSubmit={this.handleSubmit}>
             <Form.Row>
               <Col>
+              <Col>
+                <Form.Group controlId="formIndexInput">
+                  <Form.Label>Index name</Form.Label>
+                  <Form.Control
+                    type="text"
+                    value={this.props.filter_index}
```

Comment:
I think that inverted here

Reply:
I the UI I can type in "Repository regexp". I guess due to that.

Reply:
fixed